src/BugsnagServiceProvider.php

```diff
@@ -30,6 +30,19 @@ public function boot()
         }
 
         $this->mergeConfigFrom($source, 'bugsnag');
+
+        $callback = function () {
+            $this->app['bugsnag']->flush();
+        };
+
+        $this->app['queue']->after($callback);
+        $this->app['queue']->stopping($callback);
+
+        if (method_exists($this->app['queue'], 'exceptionOccurred')) {
```

Comment:
This method does not exist on L5.0 or L5.1, and the after event is not fired when the job throws an exception, so we need to flush events on looping instead on earlier versions.